src/ralph/backends/http/async_lrs.py

```diff
@@ -35,10 +40,31 @@ class StatementResponse(BaseModel):
     more: Optional[str]
 
 
+class LRSStatementsQuery(BaseModel):
+    """Pydantic model for LRS query on Statements resource query parameters."""
+
+    # pylint: disable=too-many-instance-attributes
+
+    statementId: Optional[str] = None  # pylint: disable=invalid-name
+    voidedStatementId: Optional[str] = None  # pylint: disable=invalid-name
+    agent: Optional[Union[BaseXapiAgent, BaseXapiGroup]] = None
+    verb: Optional[IRI] = None
+    activity: Optional[IRI] = None
+    registration: Optional[UUID] = None
+    related_activities: Optional[bool] = False
+    related_agents: Optional[bool] = False
+    since: Optional[datetime] = None
+    until: Optional[datetime] = None
+    limit: Optional[NonNegativeInt] = 0
+    format: Optional[Literal["ids", "exact", "canonical"]] = "exact"
+    attachments: Optional[bool] = False
+    ascending: Optional[bool] = False
+
+
 class LRSQuery(BaseQuery):
     """LRS body query model."""
 
-    query: Optional[dict]
+    query: Optional[LRSStatementsQuery]
```

Comment:
Do we need this level of indirection? Why not using the `LRSStatementQuery` and remove the `LRSQuery` model?